docs/components/categories.rst

```diff
@@ -1,4 +1,73 @@
 Categories
 ==========================================================
 
+Categories are a way to organize Mautic elements.
+Mautic has a ``CategoryBundle`` that you can leverage to incorporate Categories into your Plugin.
 
+.. vale off
+
+Adding Categories
+-----------------
+
+.. vale on
+
+You can add Categories through your Plugin's ``config.php`` file by adding the following as a key to the returned config array:
+
+.. code-block:: php
+
+    <?php
+
+    'categories' => [
+        'plugin:helloWorld' => 'mautic.helloworld.world.categories'
+    ]
+
+Please prefix Category keys with ``plugin:`` it determines permissions to manage Categories.
+The ``helloWorld`` should match the permission class name.
+
+.. vale off
+
+Configuring Categories for Routes
+---------------------------------
+
+.. vale on
+
+There is no need to add custom routes for Categories.
+However, when generating a URL to the Plugin's Category list, use the following code:
+
+.. code-block:: php
+    
+    <?php
+
+    $categoryUrl = $router->generateUrl('mautic_category_index', ['bundle' => 'plugin:helloWorld']);
+
+.. vale off
+
+Including Category in Forms
```

Comment:
```suggestion
Including Categories in Forms
```
If you're referring to singular then 'Including a Category in Forms', but I think we're referring to the plural?